c_api/gpu/GpuIndex_c.cpp

```diff
@@ -8,9 +8,64 @@
 // -*- c++ -*-
 
 #include "GpuIndex_c.h"
-#include <faiss/gpu/GpuIndex.h>
+#include "GpuAutoTune_c.h"
 #include "macros_impl.h"
+#include <faiss/gpu/GpuIndex.h>
+#include <faiss/gpu/GpuIndexCagra.h>
+#include <faiss/gpu/StandardGpuResources.h>
+#include <faiss/impl/FaissAssert.h>
 
 using faiss::gpu::GpuIndexConfig;
 
 DEFINE_GETTER(GpuIndexConfig, int, device)
+
+int faiss_index_gpu_to_cpu_new(const FaissIndex* gpu_index, FaissIndex** p_out) {
+    int result = faiss_index_gpu_to_cpu(gpu_index, p_out);
```

Comment:
What would be the difference between `faiss_index_gpu_to_cpu_new` and `faiss_index_gpu_to_cpu`? If these function are to ever have a different behavior, then that should probably be documented.